src/lib/src/component/navigation/navigation.component.test.setup.ts

```diff
@@ -10,6 +10,9 @@ import { PersistenceService } from '../../service/persistence/persistence.servic
 import { TestExecutionService } from '../../service/execution/test.execution.service';
 import { Response, ResponseOptions } from '@angular/http';
 
+export const HTTP_STATUS_CREATED = 201;
```

Comment:
we should probably think about having a central include/component with such definitions (if not provided by angular or typescript)

Reply:
There does not seem to be predefined constants provided by Angular (see e.g. https://stackoverflow.com/questions/45051382/angular-2-http-response-status-codes-as-constants). There is this: https://www.npmjs.com/package/http-status-codes